docs/howtos/DataPlane Generation - DPG/07tcgcTypes.mdx

```diff
@@ -1340,9 +1350,10 @@ export interface SdkConstantType extends SdkTypeBase {
 export interface SdkUnionType extends SdkTypeBase {
   name: string;
   // determines if the union name was generated or not
-  generatedName: boolean;
+  isGeneratedName: boolean;
```

Comment:
this is another breaking change, can you make a separate pr for this change, or at least call it out in the changeset?

Reply:
this is not a breaking change I believe.
I did not change this API in this PR. I only updated this file (a document) to align the latest APIs in our code.

Reply:
ohh i see, didn't realize it was a doc file. All good then